### PR TITLE
Add language chooser menu

### LINKED ISSUE #39: Add language chooser menu
Continued from #35 

Now that we are supporting multi-lingual static content, we want users to be able to choose the desired language from a list of supported languages. Each time we add a new language translation, we will ensure the language list is (automatically?) updated.

## Task

- [ ] add a language chooser select menu to the site navigation
- [ ] ensure that all supported languages are (automatically) included in the language chooser select menu

## Resources

- Django docs: [Switching language in templates](https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#switching-language-in-templates)

### PR DESCRIPTION
**Pull request by Upasanadhameliya**
Closes #39 

## What?
- Language choser menu/form added on `navbar` dropdown.
- On form submit, the website language changes to the selected one.

### CODE REVIEW

**Review by brylie — CHANGES_REQUESTED** [review]
This is quite a lot of languages to support from the start. Let's pick a couple of initial languages and add more incrementally. Do you speak any languages other than English? If so, would you like to provide the first localization for our project? We can enable the languages you speak and create a separate task to localize the text. Then, we can add languages based on widespread usage as well as the availability of localization help.

**Comment by Upasanadhameliya** [comment]
@brylie I am able to speak and read `Hindi (hi)` besides English. Any other language you want to keep besides this?

**Comment by brylie** [comment]
OK, cool! Let's create a follow-up task for Hindi localization :-)

File: project/core/templates/navbar.html [hunk 1]
```diff
@@ -19,6 +19,16 @@
...
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="#languageMenu" id="languageDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                    {% translate "EN" %}
+                </a>
+
+                <ul class="dropdown-menu dropdown-menu-end" id="languageMenu" aria-labelledby="languageDropdown">
+                    <li><a class="dropdown-item" href="#">{% translate "English - EN" %}</a></li>
+                    <li><a class="dropdown-item" href="#">{% translate "Hindi - HI" %}</a></li>
+                </ul>
+            </li>
```
Comment (brylie) [inline project/core/templates/navbar.html:31]: Let's use the example from the Django [set language set_language redirect view](https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#the-set-language-redirect-view) docs

```html
{% load i18n %}

<form action="{% url 'set_language' %}" method="post">{% csrf_token %}
    <input name="next" type="hidden" value="{{ redirect_to }}">
    <select name="language">
        {% get_current_language as LANGUAGE_CODE %}
        {% get_available_languages as LANGUAGES %}
        {% get_language_info_list for LANGUAGES as languages %}
        {% for language in languages %}
            <option value="{{ language.code }}"{% if language.code == LANGUAGE_CODE %} selected{% endif %}>
                {{ language.name_local }} ({{ language.code }})
            </option>
        {% endfor %}
    </select>
    <input type="submit" value="Go">
</form>
```

Make sure to read the docs carefully and remember to 
- set the `i18n` URLs
- define the list of supported languages, if needed

**Review by brylie — CHANGES_REQUESTED** [review]

**Review by brylie — APPROVED** [review]

**Pull request merged**